themes/aframe/source/css/_common.styl

```diff
@@ -226,3 +226,34 @@ h2, h3, h4
 
     .logo__wordmark
         margin-bottom 20px
+
+.sponsor {  
+  vertical-allign: middle;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  margin: 0 20px 20px;
+  height: 30px;
+}
+.sponsor .message {
+  margin-right: 7px;
+  display: inline-block;
+  height: 30px;
+  line-height: 30px;
+  vertical-align: top;
+  font-weight: 300;
+}
+a.mozilla-logo {
+  display: inline-block;
+  opacity: 0.8;
+  background: url(../images/mozilla-logo.svg) no-repeat;
+  height: 24px;
+  width: 88px;
```

Comment:
can you change this to `84px`? (it's the correct proportional width so the logo fits snuggly to the edges.)